src/components/generics/vacation-test-forms/employeeVacationRequests.tsx

```diff
@@ -99,7 +100,7 @@ const ExpandableRow = (
         <StyledTableCell>{ startDate }</StyledTableCell>
         <StyledTableCell>{ endDate }</StyledTableCell>
         <StyledTableCell>{ remainingDays }</StyledTableCell>
-        <StyledTableCell sx={{ "&.pending": { color: "#FF493C" }, "&.accepted": { color: "#45cf36" } }} className={status === "ACCEPTED" ? "accepted" : "pending"}>{ status }</StyledTableCell>
+        <StyledTableCell sx={{ "&.pending": { color: "#FF493C" }, "&.approved": { color: "#45cf36" } }} className={status === "APPROVED" ? "approved" : "pending"}>{ status }</StyledTableCell>
```

Comment:
It seems to me like there is some repetition of the styles going on here with the color being referenced in the StyledTableCell and then done inline here as well. I'm pretty rusty on the best practice for setting styles, lets try to have a chat about what is going on here at some point even if we don't actually change anything.